src/app/issue/issue-assigned/issue-assigned.component.ts

```diff
@@ -7,7 +7,7 @@ import { tap } from "rxjs/operators";
 import { ProjectUser } from "../../interfaces/project/ProjectUser";
 import { ProjectUserService } from "../../project/project-user.service";
 import { SubscriptionWrapper } from "../../SubscriptionWrapper";
-import * as Identicons from 'identicon.js';
+//import * as Identicons from 'identicon.js';
```

Comment:
Hier müssten die Auskommentierten Code Zeilen auch wieder in den Code.

Wenn diese Identicons bei dir Probleme machen, dann versuch mal im terminal 'npm install' auszuführen